rest-service/manager_rest/storage/storage_manager.py

```diff
@@ -605,7 +643,14 @@ def refresh(self, instance):
         self._load_relationships(instance)
         return instance
 
+    @traces()
```

Comment:
This will work when this func was not called before?

Reply:
I don't understand the question or more specifically why it's possible for it _not_ to work?